commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -40,6 +40,15 @@ def is_edx_lms_order(order: CTOrder) -> bool:
     return len(get_edx_items(order)) >= 1
 
 
+def get_course_mode_from_ct_order(line_item: CTLineItem) -> str:
+    mode = 'verified'
+    for attribute in line_item.variant.attributes:
+        if attribute.name == 'mode':
+            mode = attribute.value
+
+    return mode
```

Comment:
We already have a function that does the same thing. (Ref: https://github.com/edx/commerce-coordinator/blob/3f583c0d71eb89b316dfa8e6266e536481bfe410/commerce_coordinator/apps/commercetools/sub_messages/tasks.py#L205).

Can we make it a utility and use it here for our use case?

Reply:
Once this is addressed, approved from my side.

Reply:
I did it